### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by jordaaash**
This PR is intended to fix some dev/peer/dependencies and version discrepancies.

### CODE REVIEW

File: js/yarn.lock [hunk 1]
```diff
@@ -2410,53 +2403,61 @@
...
-  version "1.1.0"
-  resolved "https://registry.yarnpkg.com/@solana/wallet-standard-util/-/wallet-standard-util-1.1.0.tgz#e19d91dae0e84b56724bd132abf1cef4a1824324"
-  integrity sha512-vssoCIx43sY5EMrT1pVltsZZKPAQfKpPG3ib2fuqRqpTRGkeRFCPDf4lrVFAYYp238tFr3Xrr/3JLcGvPP7uYw==
+"@solana/wallet-standard-features@^1.2.0":
+  version "1.2.0"
+  resolved "https://registry.yarnpkg.com/@solana/wallet-standard-features/-/wallet-standard-features-1.2.0.tgz#be8b3824abf5ebcfeaa7298445bf53f76a27c935"
+  integrity sha512-tUd9srDLkRpe1BYg7we+c4UhRQkq+XQWswsr/L1xfGmoRDF47BPSXf4zE7ZU2GRBGvxtGt7lwJVAufQyQYhxTQ==
+  dependencies:
+    "@wallet-standard/base" "^1.0.1"
+    "@wallet-standard/features" "^1.0.3"
```
Comment (jordaaash) [inline js/yarn.lock:2412]: So this isn't ideal, we're ending up with two versions here...

**Comment by Funkatronics** [comment]
apart from the duplicate wallet-standard-features dep, this LGTM. builds and runs fine for me locally.

**Comment by Funkatronics** [comment]
@jordaaash I am going to cherry pick these changes and merge them in, leaving out the react-native-walletlib changes for now since that is a separate lib and I want Mikes input before I go mucking around in there.

**Comment by Funkatronics** [comment]
@jordaaash I am going to cherry pick these changes and merge them in, leaving out the react-native-walletlib changes for now since that is a separate lib and I want Mikes input before I go mucking around in there.

**Pull request closed without merging**